doc/src/conf.py

```diff
@@ -30,6 +30,14 @@
               'sphinx.ext.mathjax', 'numpydoc', 'sympylive',
               'sphinx.ext.graphviz', 'matplotlib.sphinxext.plot_directive']
 
+redirects = {
+    "install.rst": "getting_started/install.rst",
+    "aboutus.rst": "getting_started/aboutus.rst",
```

Comment:
This has a list of contributors (not sure if this matches the AUTHORS file), the license, and a list of financial support. I think the list of contributors and financial support better on the sympy website, instead of the docs.